README.md

```diff
@@ -51,6 +51,77 @@ npm install
 truffle test
 ```
 
+## Truffle interactive blockchain console
+
+To interact with deployed contracts you can use the truffle console:
+
+```
+MNEMONIC_PHRASE=$MNEMONIC RPC_OPTIMISM_GOERLI=$RPC_OPTIMISM_GOERLI RPC_POLYGON_MUMBAI=$RPC_POLYGON_MUMBAI truffle console --network=mumbai
+```
+
+From the console you can then call "migrate" which will execute the code in `migrations/1_initial_migration.js` and deploy the respective contracts to the selected blockchain. From the console you can instantiate a reference to the contract using:
+
+```javascript
+let instance = await GNFT.at(
+  "0xF3f95d49D5205d0aD811ebdadA1c106aef7D69f2"
+);
+```
+
+```shell
+truffle(mumbai)> instance.initialize("", "1000000000", 300)
```

Comment:
@DeusExM4ch1n4 Why the empty string for the first argument here?

Reply:
This is a reused example from the old docs in Share ReadMe. It is quite outdated as the pfa initialization accepts many more arguments now. I will update it to a more accurate example.